board/batocera/dropbear_localoptions.h

```diff
@@ -1 +1,6 @@
 #define DEFAULT_RECV_WINDOW 10485760
+
+#define DSS_PRIV_FILENAME "/userdata/system/ssh/dropbear_dss_host_key"
```

Comment:
Where are these new defines used?

Reply:
See:

https://github.com/batocera-linux/batocera.linux/blob/3ddf88ef7da39d350dd1b03d4714dad4d8eb3a03/package/batocera/core/batocera-system/Config.in#L173

https://github.com/batocera-linux/batocera.linux/blob/3ddf88ef7da39d350dd1b03d4714dad4d8eb3a03/configs/batocera-board.common#L93

https://github.com/batocera-linux/buildroot/blob/09eab9750c419237d3f85f015416a6c0567759eb/package/dropbear/dropbear.mk#L122

Reply:
OK, so since the /etc/dropbear symlink now points to /userdata/system/.ssh, and I assume these paths are for system-wide keys, then the default dropbear paths need to be overridden which is what happens here.

Reply:
Yes, with "now" referring to the situation after the PR is applied.